### PR TITLE
Add configuration for the Chromium browser

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

**Comment by jugglinmike** [comment]
Having second thoughts about attempting to update the browser with every invocation. Updating really ought be an explicit operation that is explicitly triggered. I've modified the `chromium-latest` script to simply run whatever has been installed instead.

**Comment by jugglinmike** [comment]
And of course, we need to forward arguments for that script to be useful at all.

File: roles/chromium/templates/chromium-latest [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+{{playbook_dir}}/chromium-latest-linux/run.sh $*
```
Comment (bobholt) [inline roles/chromium/templates/chromium-latest]: @jugglinmike This only works if you've run the `update.sh` at least once. That is, unless you run either `update-and-run` or `update` first, `run.sh` doesn't do anything. I know you changed this so it doesn't update every time, but maybe there should be an initial call to update in the ansible script?

Comment (jugglinmike) [inline roles/chromium/templates/chromium-latest]: Good catch! I agree :100:% Just pushed up a commit to prove it.

**Comment by bobholt** [comment]
:+1: closed via 27798275d50967179440e9ffec63a79f373a0958

**Comment by jugglinmike** [comment]
Thanks!

**Pull request closed without merging**